geoh5py/objects/object_base.py

```diff
@@ -562,34 +601,7 @@ def remove_property_group(self, property_group: PropertyGroup):
             and property_group in self._property_groups
         ):
             self._property_groups.remove(property_group)
-
-    @property
-    def vertices(self) -> np.ndarray:
-        r"""
-        :obj:`numpy.array` of :obj:`float`, shape (\*, 3): Array of x, y, z coordinates
-        defining the position of points in 3D space.
-        """
-        raise AttributeError(
-            f"Object {type(self)} does not have the attribute 'vertices'."
-        )
-
-    @property
-    def locations(self):
-        """Exposes the vertices or centroids of the object."""
-        out = None
-        if hasattr(self, "vertices"):
-            out = self.vertices
-        if hasattr(self, "centroids"):
-            out = self.centroids
-
-        return out
-
-    @property
-    def converter(self):
-        """
-        :return: The converter for the object.
-        """
-        return self._converter
+            # todo: this should suppress the property group object too? del property_group
```

Comment:
Either make the `remove_property_group` method private, or move that little piece of logic back inside the `remove_children`, so that the PropertyGroup is guaranteed to be deleted by the workspace.